build-system/compile/bundles.config.bento.json

```diff
@@ -2,6 +2,8 @@
   {
     "name": "jwplayer",
     "version": "1.0",
-    "options": {}
+    "options": {
```

Comment:
Does this need `bento: true` instead?

All `amp-*.css` files should be built without this flag set. If they're not being built for `bento-*` outputs, that might be a regression.

Reply:
No, bundles defined in this file always have `bento: true` as defined [here](https://github.com/ampproject/amphtml/blob/22fb109b134123bb1c03db0f0f8b3061505040ca/build-system/tasks/build-bento.js#L138)

Reply:
Are the other `.css` files output? Would love to get rid of the css flag.

Reply:
I misunderstood, this handles only one extension.